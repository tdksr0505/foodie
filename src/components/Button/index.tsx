import styled from 'styled-components';
import Button from '@mui/material/Button';
export default styled(Button)`
  border: 0;
  outline: none;
  width: auto;
  height: 35px;
  padding: 2px 10px;
  background-color: #dda52e;
  color: #fff;
  font-size: 15px;
  &:hover {
    background-color: #ddbb70;
  }
`;
