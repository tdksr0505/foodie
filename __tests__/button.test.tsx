import renderer from 'react-test-renderer';
import Button from '../src/components/Button';

it('Button Component snapshop', () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});
