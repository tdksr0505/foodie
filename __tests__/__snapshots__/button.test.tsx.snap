// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component snapshop 1`] = `
<button
  className="sc-d313a61b-0 AjtaS"
/>
`;
